src/Tests/ControlTests/testoutputs/ServerSideStyleTests.PostbackHandlers.html

```diff
@@ -2,16 +2,16 @@
 	<head></head>
 	<body>
 		One handler
-		<input onclick="dotvvm.postBack(this,[],&quot;8UxwOb8axY9NSuJp&quot;,&quot;&quot;,null,[[&quot;confirm&quot;,{message:&quot;a&quot;}],&quot;validate-root&quot;],[],undefined).catch(dotvvm.log.logPostBackScriptError);event.stopPropagation();return false;" type="button" value="">
+		<input onclick="dotvvm.postBack(this,[],&quot;8UxwOb8axY9NSuJp&quot;,&quot;&quot;,null,[[&quot;confirm&quot;,{message:&quot;a&quot;}],[&quot;validate&quot;, {path:&quot;/&quot;}]],[],undefined).catch(dotvvm.log.logPostBackScriptError);event.stopPropagation();return false;" type="button" value="">
```

Comment:
This is exactly why I'd like to keep the validate-root shortcut...

Reply:
I added the shortcut in `b812885`. Although it seems like the file was changed in `main` and no longer contains `validate-root`... Regardless, it is back so I am marking as resolved.